httperrors/httperrors.go

```diff
@@ -125,3 +125,8 @@ func NewUnauthorizedError(errorName string, msg string) HTTPError {
 		true,
 	)
 }
+
+// A constructor for an HTTPError "Bad Request"
+func NewBadRequestError(err string, msg string) HTTPError {
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.